app/build.gradle

```diff
@@ -118,6 +120,9 @@ ext {
 
 dependencies {
 
+    // Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$dependencyVersions.kotlin"
```

Comment:
I think it would make sense to use `org.jetbrains.kotlin:kotlin-stdlib-jre8` if we are targeting jdk 8